src/common/mockConstants/mockSession.ts

```diff
@@ -4,4 +4,7 @@ import { mockProfile } from "./mockProfile";
 export const mockSession: SessionState = {
   isLoggedIn: true,
   profile: mockProfile,
+  idenfy: {
+    authToken: "",
+  },
```

Comment:
I thought we should put the authToken in state so we dont trigger an api call to fetch a token every time they open/close the modal? We can set the expiry time of the session/token in the admin settings as well